src/RestClient/endpoints/V3/Themes/ThemeCustomTemplates.ts

```diff
@@ -0,0 +1,24 @@
+import { AxiosInstance } from 'axios';
+
+import { themesPath } from './Themes';
+import type { ThemeCustomTemplate } from './types';
+
+class ThemeCustomTemplates {
+  private client: AxiosInstance;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+  }
+
+  /**
+   * Enumarates available custom templates for in the theme files in a specific theme version
+   *
+   * @param versionUUID valid store theme version UUID
+   * @returns Promsie resolving to a single custom theme template
```

Comment:
```suggestion
   * @returns Promise resolving to a list of available custom template files
```

Reply:
(I assume this is a better description, feel free to correct me)

Reply:
Updated description